kube/overlays/stable/kustomization.yaml

```diff
@@ -21,6 +21,8 @@ images:
     newTag: 1.7.0
   - name: airbyte/cron
     newTag: 0.40.18
+  #- name: airbyte/connector-builder #FIXME: Uncomment this block when enabling airbyte-connector-builder
+  #  newTag: 0.40.18
```

Comment:
If it is not a huge lift to also add this server to our airbyte-cloud deployment, it would be pretty nice to do that so people could just access the Connector Builder through a URL instead of having to spin up Airbyte themselves.

Alternatively, I believe the [demo.airbyte.io](demo.airbyte.io) instance is actually an OSS deployment that is just made publicly accessible. So people could access the Connector Builder there too, which has the added benefit of not requiring any authentication to use

Reply:
there are a few open questions to answer like whether we need to enforce rate limits to protect the backend against a bad user, but using the demo instance is a good idea for making the connector public without putting prod at risk